docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '3'
+services:
+
+  web:
+    image: httpd:2.4-alpine
+    volumes:
+      - ./web:/usr/local/apache2/htdocs
+      - ./docker/web/httpd.conf:/usr/local/apache2/conf/httpd.conf
+    labels:
+      - traefik.http.routers.my-container.rule=Host(`wordpress-scaffold.localhost`)
+
+  php:
+    build:
+      context: docker/php
+    working_dir: /var/www/html
+    volumes:
+      - .:/var/www/html
+    environment:
+      DB_HOST: db
+      DB_NAME: wordpress
+      DB_USER: wordpress
+      DB_PASSWORD: secret
+      WP_HOME: http://wordpress-scaffold.localhost
```

Comment:
Just curious: how does this work along with the `.env` and the stuff that gets prefilled there during the setup task?

I assume this is what's being used, right? In that case it makes those questions a bit redundant (plus those entries in the `.env` as well).

Reply:
`.env` won't overwrite OS level env vars so these vars will be used. It's not possible to set `WP_HOME` during setup, because Treafik needs that information. Because `composer setup` must be executed in the PHP container the DB_ env vars are already set. Maybe the questions could check for already set env vars.

Reply:
I've changed ProjectSetup so it will use the result of `getenv` by default.